### PR TITLE
fix: OPTIC-979: converting stylus files to scss

### PR DESCRIPTION
**Pull request by carly-bartel**
converting stylus files to scss in LS web/apps/labelstudio/src/components/

### CODE REVIEW

File: web/apps/labelstudio/src/components/Card/Card.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  &__header {
+    display: flex;
+    height: 48px;
+    padding: 0 15px;
+    align-items: center;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 18px;
+    justify-content: space-between;
+    box-shadow: 0 1px 0 0 rgba(#000, 0.1);
```
Comment (yyassi-heartex) [inline web/apps/labelstudio/src/components/Card/Card.scss]:
```suggestion
    box-shadow: 0 1px 0 0 rgba(0, 0, 0, 0.1);
```

File: web/apps/labelstudio/src/components/Form/Form.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+.input,
+.textarea,
+.counter,
+.select__list {
+
+  &:not(&_ghost):focus,
+  &_focused {
+    outline: none;
+    box-shadow: 0px 0px 0px 6px var(--grape_100), inset 0px -1px 0px rgba(#000, 0.1), inset 0px 0px 0px 1px rgba(#000, 0.15), inset 0px 0px 0px 1px var(--grape_100);
```
Comment (yyassi-heartex) [inline web/apps/labelstudio/src/components/Form/Form.scss]:
```suggestion
    box-shadow: 0px 0px 0px 6px var(--grape_100), inset 0px -1px 0px rgba(0, 0, 0, 0.1), inset 0px 0px 0px 1px rgba(0, 0, 0, 0.15), inset 0px 0px 0px 1px var(--grape_100);
```

File: web/apps/labelstudio/src/components/Pagination/Pagination.scss [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    --button-color: var(--sand_500);
+    background-color: var(--sand_200);
+    pointer-events: none;
+  }
+
+  &__label {
+    padding-right: 16px;
+    font-size: 16px;
+    line-height: 19px;
+    color: rgba(#000, 0.4);
```
Comment (yyassi-heartex) [inline web/apps/labelstudio/src/components/Pagination/Pagination.scss]:
```suggestion
    color: rgba(0, 0, 0, 0.4);
```

File: web/apps/labelstudio/src/components/SidebarMenu/SidebarMenu.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  &__navigation {
+    width: calc(var(--menu-sidebar-width) + 1px);
+    display: flex;
+  }
+
+  &__content {
+    flex: 1;
+    overflow: auto;
+    padding: 2rem;
+    box-shadow: -1px 0 0 0 rgba(#000, 0.1);
```
Comment (yyassi-heartex) [inline web/apps/labelstudio/src/components/SidebarMenu/SidebarMenu.scss]:
```suggestion
    box-shadow: -1px 0 0 0 rgba(0, 0, 0, 0.1);
```

File: web/apps/labelstudio/src/components/Button/Button.scss [hunk 5]
```diff
@@ -0,0 +1,348 @@
+.button {
+  --button-color: var(--grape_500);
+  --button-background-color: var(--sand_0);
+  --button-background-image: none;
+  --button-shadow: inset 0px -1px 0px rgba(#000, 0.1), inset 0px 0px 0px 1px rgba(#000, 0.15);
```
Comment (yyassi-heartex) [inline web/apps/labelstudio/src/components/Button/Button.scss]:
```suggestion
  --button-shadow: inset 0px -1px 0px rgba(0, 0, 0, 0.1), inset 0px 0px 0px 1px rgba(0, 0 ,0 , 0.15);
```

File: web/apps/labelstudio/src/components/Button/Button.scss [hunk 6]
```diff
@@ -0,0 +1,348 @@
...
+  &_look_danger:disabled,
+  &_look_danger &_waiting,
+  &_look_danger &_disabled {
+    --button-color: var(--sand_500);
+  }
+
+  &_look_destructive:disabled,
+  &_look_destructive &_waiting,
+  &_look_destructive &_disabled {
+    --button-color: rgba(#fff, 0.5);
```
Comment (yyassi-heartex) [inline web/apps/labelstudio/src/components/Button/Button.scss]:
```suggestion
    --button-color: rgba(255, 255, 255, 0.5);
```

**Review by yyassi-heartex — APPROVED** [review]

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10385920362)
> Successfully merged:  27 files changed, 674 insertions(+), 121 deletions(-)

**Review by AndrejOros — APPROVED** [review]

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10389432348)
> Successfully merged:  52 files changed, 1756 insertions(+), 1492 deletions(-)

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10390118221)
> Successfully merged:  53 files changed, 1705 insertions(+), 1424 deletions(-)

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10390244894)
> Successfully merged:  7 files changed, 22 insertions(+), 4 deletions(-)

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10390561538)
> Successfully merged: Already up to date.

**Pull request merged**